tests/unit/test_sqlite_edge_cases.py

```
@@ -172,7 +176,9 @@ async def test_backup_with_long_filename(self, tmp_path: Path) -> None:
         assert len(backup_files) == 1
 
     @pytest.mark.asyncio
-    async def test_backup_with_no_write_permissions(self, tmp_path: Path) -> None:
+    async def test_backup_with_no_write_permissions(
+        self, tmp_path: Path, sqlite_backend_factory
```

Ruff (0.13.2)

Unused method argument: `sqlite_backend_factory`

(ARG002)